quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/README.md

```diff
@@ -0,0 +1,129 @@
+# Create Azure NetApp Files resource for SAP-HANA Storage
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/PublicLastTestDate.svg)
```

Comment:
would be great to move all references to end of the Readme inside "Reference" section

Reply:
These are not references. I observed most of the template has the following images at the beginning. Moreover pipeline also requires to have image links.

Reply:
They are required at the top for samples in the repo